compiler/rustc_driver/src/lib.rs

```diff
@@ -961,10 +961,7 @@ Available lint options:
 
     match (loaded_plugins, plugin.len(), plugin_groups.len()) {
         (false, 0, _) | (false, _, 0) => {
-            println!(
-                "Compiler plugins can provide additional lints and lint groups. To see a \
-                      listing of these, re-run `rustc -W help` with a crate filename."
-            );
+            println!("Compiler plugins can provide additional lints and lint groups.");
```

Comment:
let's actually just get rid of this? It's not relevant to clippy, and compiler plugins are not a common thing.

Reply:
I reworded it in e54c060. Removing this would also involve dealing with this case in the match statement somehow. I think printing a message, that tools can provide more lints is a good way to deal with this case.